agir/front/api_views.py

```diff
@@ -8,6 +8,18 @@
 from agir.groups.models import SupportGroup
 from agir.groups.serializers import SupportGroupDetailSerializer
 from agir.events.serializers import EventSerializer, EventListSerializer
+from django.utils import timezone
+import json
+from django.conf import settings
+
+
+CERTIFIED = "CERTIFIED"
+NOT_CERTIFIED = "NOT_CERTIFIED"
+ALPHA_ASC = "ALPHA_ASC"
+ALPHA_DESC = "ALPHA_DESC"
+DATE_ASC = "DATE_ASC"
+DATE_DESC = "DATE_DESC"
+PAST = "PAST"
```

Comment:
À mettre plutôt dans `SearchSupportGroupsAndEventsAPIView` et avec des noms plus explicites : 
```suggestion
GROUP_FILTER_CERTIFIED = "CERTIFIED"
GROUP_FILTER_NOT_CERTIFIED = "NOT_CERTIFIED"
SORT_ALPHA_ASC = "ALPHA_ASC"
SORT_ALPHA_DESC = "ALPHA_DESC"
SORT_EVENT_END_DATE_ASC = "DATE_ASC"
SORT_EVENT_END_DATE_DESC = "DATE_DESC"
EVENT_FILTER_PAST = "PAST"
```
Mais on peut aussi ne pas en faire des variables du tout et les utiliser directement dans les fonctions (comme "groupType", "groupSort", "eventType", ...)
Pour quelque chose de plus RESTful, on pourrait aussi utiliser directement les noms des propriétés par lesquelles on veut filter/ordonner (pour les filtres il y a des exemples dans [la doc de rest framework](https://www.django-rest-framework.org/api-guide/filtering/), et pour le sort [ici](https://www.moesif.com/blog/technical/api-design/REST-API-Design-Filtering-Sorting-and-Pagination/#sorting))

Reply:
J'ai opté pour le renommage du coup, cela me paraît clair comme ça et que les filtres sont plus complexes que des équivalences d'attributs :)